src/game/gamecore.h

```diff
@@ -36,7 +37,12 @@ class CTuningParams
 
 	static const char *m_apNames[];
 
-	#define MACRO_TUNING_PARAM(Name,ScriptName,Value) CTuneParam m_##Name;
+	#define MACRO_TUNING_PARAM(Name,ScriptName,Value) CTuneParam m_##Name;\
+	float GetTuneI_##Name() const { return m_##Name.Get(); } \
+	float GetTuneF_##Name() const { return m_##Name.GetLua(); } \
+	void SetTuneI_##Name(int v) { m_##Name.Set(v); }\
+	void SetTuneF_##Name(float v) { m_##Name.Set((int)v*100); }
```

Comment:
Casting `v` to int sacrifices precision, e.g. if `v = 0.9f` then `(int)v*100` returns 0, although 90 would be expected.

You can use this instead:
```cpp
m_##Name.Set(round_to_int(v*100.0f));
```

Reply:
...and while you're at it, please indent these 4 lines belonging to the macro definition with 2 additional tabs to make it clear that they belong to the macro.